src/main/java/io/outfoxx/swiftpoet/CodeWriter.kt

```diff
@@ -376,7 +379,9 @@ internal class CodeWriter(
    */
   private fun resolveImport(typeName: DeclaredTypeName): String {
     val topLevelTypeName = typeName.topLevelTypeName()
```

Comment:
Maybe add a `val topLevelSimpleName = topLevelTypeName.simpleName` since you need it a few times?